src/ark/phenotyping/pixie_preprocessing.py

```diff
@@ -282,11 +328,41 @@ def create_pixel_matrix(fovs, channels, base_dir, tiff_dir, seg_dir,
         channels=channels
     )
 
+    # load existing channel_norm_path if exists, otherwise generate
+    if not os.path.exists(channel_norm_path):
+        # compute channel percentiles
+        channel_norm_df = pixel_cluster_utils.calculate_channel_percentiles(
+            tiff_dir=tiff_dir,
+            fovs=fovs,
+            channels=channels,
+            img_sub_folder=img_sub_folder,
+            percentile=channel_percentile_postnorm
```

Comment:
I think `percentile` here should be a new variable called `channel_percentile_prenorm`. `channel_percentile_postnorm` is the percentile we are using to calculate the normalization values after pixel-level row normalization. This came up in Cami's pull request last time - we noticed that we were using 99% for the pre-row normalization, and 99.9% for the post-row normalization. I think I asked @ngreenwald awhile ago why he used 99 and not 99.9 and I think the answer was just that it looked better for his cohort. So I think solution for most flexibility is to have one more input variable (`channel_percnetile_prenorm`) to `create_pixel_matrix` and set it to 0.99 as a default.

Reply:
So just to clarify did you want to rename the actual parameter argument `channel_percentile_prenorm` or just assign `percentile=channel_percentile_prenorm` instead?

Reply:
Assigning `percentile=channel_percentile_prenorm` is fine